app/client/reducers/reducer_groupMessages.js

```diff
@@ -0,0 +1,34 @@
+import {
+  SET_GROUP_MESSAGES,
+  FETCHING_GROUP_MESSAGES,
+  FETCH_GROUP_MESSAGES_FAILURE } from '../actions/types';
+
+const initialState = {
```

Comment:
Put all your initial states for your reducers in some sort of initialState.js or any file name of your choosing